thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -593,7 +562,7 @@ export default Controller.extend({
         'selectedMetricOption'
       );
       const pattern = selectedPattern ? optionMap.pattern[selectedPattern] : null;
-      const formattedPattern = pattern ? `${pattern.toLowerCase().replace(',',' ').camelize()}_` : '';
+      const formattedPattern = pattern ? `${pattern.toLowerCase().replace(',', ' ').camelize()}_` : '';
```

Comment:
Nit: Here we can set the default to an empty string instead of `null`  
```
  const pattern = selectedPattern ? optionMap.pattern[selectedPattern] : '';
``` 
above so that we can just write 
```
const formattedPattern = pattern.toLowerCase().replace(',', ' ').camelize()
```

In my opinion, ^ is more readable.

Reply:
Good suggestion. However, I want to avoid formatting it if pattern is empty - don't want to end up with an extra `_` in the string.